Add improved Filament Guide with ECAS04 pneumatic fitting mount

lcolok opened the pull request:
## Summary

This PR introduces a structural improvement to the Filament Guide component.
The goal is to address an issue where the PTFE tube can slip out during spooling,
thereby improving the reliability of the filament path and the stability of the respooling process.

## Problem Description

In the current design, the PTFE tube is only held in place by friction within the Filament Guide.
During operation, the combination of filament tension and tube friction can cause the PTFE tube to gradually slide out,
leading to feeding resistance, belt slippage, or stepper motor overload, and eventually interrupting the spooling process.

## Proposed Solution

To address this issue, the front section of the Filament Guide has been modified internally
to include an embedded mount for an **ECAS04 Bowden pneumatic fitting** (compatible with 4 mm OD PTFE tube).
This allows the PTFE tube to be mechanically locked to the guide, preventing it from coming loose during operation.

  <table>
    <tr>
      <td width="50%">
        <img src="https://github.com/user-attachments/assets/b1b6da25-7076-44a1-bcc7-bea8e55b7f6f" alt="Perspective view">
        <p align="center"><b>Perspective view showing the internal cavity</b></p>
      </td>
      <td width="50%">
        <img src="https://github.com/user-attachments/assets/650fbc04-c3b4-4db3-9c0e-11a803f5704e" alt="Detail view">
        <p align="center"><b>Close-up of ECAS04 fitting mount</b></p>
      </td>
    </tr>
  </table>

### Implementation Details

- An internal cavity has been integrated within the existing outlet channel, allowing the ECAS04 fitting to be directly screwed in and secured
- This change is **fully internal** — the external geometry of the Filament Guide remains identical to the original design, ensuring no interference with other components
- The new Filament Guide must be reprinted and used as a direct replacement, but it remains **fully compatible** with the original:
  - Filament Switch Stopper
  - Top cover
  - Mounting holes
- All dimensional details and fitting geometry are documented in the attached STEP file

## Notes and Trade-offs

This modification is proposed as a conceptual mechanical improvement.

**Pros:**
- Significantly increases the retention strength of the PTFE tube
- Prevents operational interruptions due to tube slippage
- Maintains full compatibility with existing parts

**Cons:**
- Introduces a slight increase in BOM cost (ECAS04 fitting)
- Alternative solution: A self-tapping screw could be used to lightly clamp the PTFE tube in place (less elegant but achieves similar functionality at lower cost)

## Files Updated

- `3D Files/Filament Guide (ECAS04 Mount).step`
  - Added embedded ECAS04 pneumatic fitting mount
  - Maintains full compatibility with existing Switch Stopper, top cover, and mounting geometry
  - Can be directly printed and used to replace the original Filament Guide

## To the Author

This modification is shared as a potential improvement idea for your consideration.
It aims to enhance the PTFE tube retention without affecting the existing assembly.
If you find this concept useful, please feel free to evaluate its cost and mechanical suitability.
Detailed dimensions and integration geometry can be found in the attached STEP file.

lcolok committed:
Add improved Filament Guide with ECAS04 mount to Extras

This community variant includes an embedded ECAS04 pneumatic fitting mount to prevent PTFE tube slippage during operation. The modification addresses reliability issues where friction alone was insufficient to retain the tube under filament tension.

Key improvements:
- Internal cavity for ECAS04 Bowden fitting (4mm PTFE)
- Mechanical locking prevents tube detachment
- External geometry unchanged (fully compatible with existing parts)
- Placed in Extras folder as a community modification

Files added:
- 3D Files/Extras/Filament Guide (ECAS04 Mount).step

LukasT03 commented:
Hi! Thanks a lot for your contribution!

I think this is a great solution to the problem (which I didn’t even realize existed)! However, I’ve decided to only use non-electronic parts that are available through Bambu Lab’s Maker’s Supply.

I’ll address this issue in the official files by slightly reducing the hole size of the lower hole in the Filament Guide.

As for your solution: Could you please upload it as a separate model on MakerWorld for those who want to use it, and also add the files to the new Extras folder I created? I’ll make sure to link to your MakerWorld model there as well.

Thank you! :)
Lukas

lcolok committed:
Add Bambu Lab print profile for ECAS04 variant

Includes optimized 3MF print profile for the Filament Guide with ECAS04 mount.
This allows users to directly import the configuration into Bambu Studio/Bambu Lab printers.

Files added:
- 3D Files/Extras/Filament Guide (ECAS04 Mount).3mf
